core/site.go

```diff
@@ -182,7 +182,8 @@ func NewSiteFromConfig(
 	}
 
 	if len(site.batteryMeters) > 0 && site.ResidualPower <= 0 {
-		site.log.WARN.Println("battery configured but residualPower is missing (add residualPower: 100 to site)")
+		site.log.WARN.Println("ResidualPower is set to 100, because battery is configured and residualPower is missing or <= 0.")
+		site.ResidualPower = 100
 	}
```

Comment:
Ich hatte so etwas in der Art im Kopf:
```suggestion
	if len(site.batteryMeters) > 0 {
		if site.ResidualPower == 0 {
			site.log.WARN.Println("ResidualPower is set to 100, because battery is configured and residualPower is missing or = 0.")
			site.ResidualPower = 100
		}

		if site.ResidualPower < 0 {
			site.log.WARN.Println("battery configured but residualPower is < 0 (add residualPower: 100 to site), see https://https://docs.evcc.io/docs/reference/configuration/site#residualpower")
		}
	}
```